conda_smithy/feedstocks.py

```diff
@@ -83,9 +83,9 @@ def clone_feedstock(feedstock_gh_repo, feedstocks_dir):
     clone_directory = os.path.join(feedstocks_dir, repo.name)
     if not os.path.exists(clone_directory):
         print('Cloning {}'.format(repo.name))
-        new_repo = Repo.clone_from(repo.clone_url, clone_directory)
-        clone.delete_remote('origin')
+        Repo.clone_from(repo.clone_url, clone_directory)
     clone = Repo(clone_directory)
+    clone.delete_remote('origin')
```

Comment:
I could also remove the remote in the if-clause then the origin remote would exist for sure.